app/components/title-field.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Icon from './icon'
+import { Plain as PlainButton, Green as GreenButton } from './button'
```

Comment:
this can be written shorter using

```js
import { * as Button }  from './button'
```

and then

```js
<Button.Green />
```

Reply:
I'd prefer not to have any `*` in the import statements as that makes it unclear which of the parts of `./button` are required in the file by just looking at the header.

Reply:
I see that point, but for buttons this is literally just different colors, so you don't get relevant information out of this

Reply:
?! Just reading it like this: I get the information what buttons are used, because only those two buttons are referenced. If I don't use one of the two buttons the linter will complain about an unused reference.

Reply:
Hm, not sure if I should go ahead with this one?

Reply:
ok, then 2>1 => PlainButton it is :)